### DISCUSSION TITLE
How old am I?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#990] [Galaxy Trivia] [answered]
On my 🎂🎈 in 2021, my age equals the sum of the digits of the year I was born.

P.S. I am not yet 💯.

**Comment by peterbaccaro** [comment]
The birth year can either be in the 1900s or the 2000s. 
Let those numbers be 19ab and 20ab and consider each case separately.

```
Sum digits = Age
sum(19ab) = 2021 – birth year
1 + 9 + a + b = 2021 – 19ab
10 + a + b= 2021 – (1900 + 10a + b)
10 + a + b = 121 – 10a – b
11a + 2b = 111
99 + 2b = 111
2b = 12
b = 6

So one possible birth year is 1996.
```

```
Sum digits = Age
sum(20ab) = 2021 – birth year
2 + 0 + a + b = 2021 – 20ab
2 + a + b= 2021 – (2000 + 10a + b)
2 + a + b = 21 – 10a – b
11a + 2b = 19
11 + 2b = 19
2b = 8
b = 4

So another possible birth year is 2014.
```

So the answer could be either **25** or **7** years old.

**Comment by thuongtruong1009** [comment]
if on your birthday in 2021, your age equals the sum of the digits of the year you were born, it means that you were born in 1982. In 2021, you would have turned 39 years old.

**Comment by CelioAmaral** [comment, marked as answer]
The birth year can either be in the 1900s or the 2000s. Let those numbers be 19ab and 20ab. Consider each case separately.

Sum digits = Age
sum(19ab) = 2021 – birth year
1 + 9 + a + b = 2021 – 19ab
10 + a + b= 2021 – (1900 + 10a + b)
10 + a + b = 121 – 10a – b
11a + 2b = 111

Since 111 is odd, and 2b is even, it must be that 11a is odd. Since a is a digit from 0 to 9, it must be an odd digit 1 to 9. This gives possibilities that 11a = 11, 22, …, 99. Consider if 11a = 77. Then we have:

77 + 2b = 111
2b = 34
b = 17 > 9

We need b to be a digit 9 or less, so this possibility has no solution. Similarly, we can eliminate 55, 33, and 11 since those will lead to even higher values of b. The only possibility that might work is 99 with a = 9. This gives:

99 + 2b = 111
2b = 12
b = 6

Thus we get a solution of 1996.

We can similarly solve for the case of 20ab.

Sum digits = Age
sum(20ab) = 2021 – birth year
2 + 0 + a + b = 2021 – 20ab
2 + a + b= 2021 – (2000 + 10a + b)
2 + a + b = 21 – 10a – b
11a + 2b = 19

Since 19 is odd, and 2b is even, it must be that 11a is odd. Since 11(3) = 33 exceeds 19 and is too large, the only possibility that could work is 11(1) = 11. This gives:

11 + 2b = 19
2b = 8
b = 4

Thus we get another solution of 2014.

The answer is the birth year could be 1996 or 2014.

On my birthday in 2021, my age is equal to the sum of the digits of my birth year. I am less than 100 years old. How old am I?

1996 age = 2021 - 1996 = 25 = 1+9+9+6

2014 age = 2021 - 2014 = 7 = 2+0+1+4

His age is equal to 25 or 7 years.
